### ISSUE TITLE
EventEmitter memory leak

### ISSUE THREAD

**Issue by dylanparry** [mobxjs/mobx-react-devtools#35]
Hi,

I’m getting the following error when I attempt to test a component that contains the Mobx React Devtools component as a child:

```
console.error E:\Documents\Visual Studio Projects\Opus4 Core\Opus.Web.Scripts\node_modules\mobx-react-devtools\index.js:1
      (node) warning: possible EventEmitter memory leak detected. 11 listeners added. Use emitter.setMaxListeners() to increase limit.
    console.error console.js:89
      Trace
          at n.addListener (E:\Documents\Visual Studio Projects\Opus4 Core\Opus.Web.Scripts\node_modules\mobx-react-devtools\index.js:1:8225)
          at t.u.value (E:\Documents\Visual Studio Projects\Opus4 Core\Opus.Web.Scripts\node_modules\mobx-react-devtools\index.js:2:4279)
          at invokeComponentDidMountWithTimer (E:\Documents\Visual Studio Projects\Opus4 Core\Opus.Web.Scripts\node_modules\react\lib\ReactCompositeComponent.js:60:18)
          at CallbackQueue._assign.notifyAll (E:\Documents\Visual Studio Projects\Opus4 Core\Opus.Web.Scripts\node_modules\react\lib\CallbackQueue.js:67:22)
          at ReactReconcileTransaction.ON_DOM_READY_QUEUEING.close (E:\Documents\Visual Studio Projects\Opus4 Core\Opus.Web.Scripts\node_modules\react\lib\ReactReconcileTransaction.js:81:26)
          at ReactReconcileTransaction.Mixin.closeAll (E:\Documents\Visual Studio Projects\Opus4 Core\Opus.Web.Scripts\node_modules\react\lib\Transaction.js:204:25)
          at ReactReconcileTransaction.Mixin.perform (E:\Documents\Visual Studio Projects\Opus4 Core\Opus.Web.Scripts\node_modules\react\lib\Transaction.js:151:16)
          at batchedMountComponentIntoNode (E:\Documents\Visual Studio Projects\Opus4 Core\Opus.Web.Scripts\node_modules\react\lib\ReactMount.js:127:15)
          at ReactDefaultBatchingStrategyTransaction.Mixin.perform (E:\Documents\Visual Studio Projects\Opus4 Core\Opus.Web.Scripts\node_modules\react\lib\Transaction.js:138:20)
          at Object.ReactDefaultBatchingStrategy.batchedUpdates (E:\Documents\Visual Studio Projects\Opus4 Core\Opus.Web.Scripts\node_modules\react\lib\ReactDefaultBatchingStrategy.js:63:19)
```

I’m attempting to mount the parent component using Enzyme's `mount` function. Is there a way of either fixing this so the error goes away, or telling the Dev Tools not to render during tests so it becomes a non-issue?

Thanks :)

**Comment by andykog** [comment]
Released fix as `mobx-react-devtools@4.2.6`
@dylanparry, could you confirm fix?

**Comment by dylanparry** [comment]
Thanks for the quick response. The error has now gone, so looks like it worked.
